module/VuFindSearch/src/VuFindSearch/Backend/EDS/Base.php

```diff
@@ -176,13 +186,13 @@ public function createSession(
             . "$profile, guest: $isGuest, authToken: $authToken "
         );
         $qs = ['profile' => $profile, 'guest' => $isGuest];
-        $url = $this->edsApiHost . '/createsession';
+        $url = $this->sessionHost . '/createsession';
```

Comment:
Would it be slightly more equivalent to the old functionality if we defaulted sessionHost to null and then used `($this->sessionHost ?? $this->edsApiHost)` here? Otherwise, if somebody has customized their API URL in the configuration but doesn't add the new session URL parameter, they'll revert to the default URL.

Reply:
I'm not sure.  I guess it depends why they are customizing the URL in the first place, which is hard to fathom, but maybe different domains for international users?  But if that's the case, I think this *should* be a breaking change, needing to add the new sessionHost property.  

Defaulting sessionHost to apiHost would work for EDS queries but not for EPF queries.  Granted that would be backwards compatible, and EDS is literally 100% of the uses of this file until this PR is merged, but after that it creates a relationship that's wrong for half the code that uses the file, and obscures the problem.

This goes back to an odd EPF API design decision to literally use the EDS path for creating a session.  If they had just created an alias for an EPF url to do that, there would be no need for the new property.  But they did, meaning it's something that needs to be handled separately by API users.  So I think VF should as well.

Reply:
That is a puzzling decision. In any case, I see your point -- I'll just add this to the changelog notes instead of insisting upon a change. :-)